docs/how-to/uc-boot-parameters.rst

```diff
@@ -0,0 +1,89 @@
+Parameters are passed as a single string in the :code:`key=value` format, with each parameter separated by a space.
+
+In the case of :doc:`real-time kernel parameters <../reference/kernel-boot-parameters>`, the most interesting parameters are `nohz`_, `nohz_full`_ and `irqaffinity`_.
+The :code:`nohz` is used to enable/disable dynamic ticks (possible values are `on` or `off`). 
+The :code:`nohz_full` receives a `cpu list`_ specifying which CPUs will have dynamic ticks disabled. 
+The `irqaffinity` is used to set the affinity of the interrupt requests (IRQs) to the CPUs, in other words: which CPUs will handle the IRQs.
+
+For example:
+
+.. code-block:: bash
+    
+    snap set system system.kernel.dangerous-cmdline-append="nohz=on nohz_full=2-N irqaffinity=0-1"
+
+
+After setting the kernel parameters, it's necessary to reboot the system to apply the changes.
+Then you can check if the parameters were applied:
+
+.. code-block:: console
+
+    $ cat /proc/cmdline
+    snapd_recovery_mode=run console=ttyS0,115200n8 console=tty1 panic=-1 nohz=on nohz_full=2-N irqaffinity=0-1
+
+
+To undo the modifications made to the kernel parameters, use the `snap unset`_ command:
+
+.. code-block:: bash
+
+    snap unset system system.kernel.dangerous-cmdline-append
+
+---
+
+Once you are satisfied with the configurations, consider configuring them statically so that they are built into the image. 
+Read more on :doc:`../how-to/uc-image-creation`.
+
+.. Is there a better way to reference an anchor on another doc? 
+
+.. LINKS
+.. _Ubuntu Core: https://ubuntu.com/core
+.. _update-grub: https://manpages.ubuntu.com/manpages/xenial/man8/update-grub.8.html
+.. _snap set: https://ubuntu.com/core/docs/modify-kernel-options
+.. _system.kernel.cmdline-append: https://snapcraft.io/docs/system-options#heading--kernel-cmdline-append
+.. _system.kernel.dangerous-cmdline-append: https://snapcraft.io/docs/system-options#heading--kernel-dangerous-cmdline-append
+.. _gadget snap: https://snapcraft.io/docs/the-gadget-snap
+.. _allow list: https://snapcraft.io/docs/the-gadget-snap#heading--dynamic
+.. _nohz: https://docs.kernel.org/timers/no_hz.html
+.. _nohz_full: https://docs.kernel.org/timers/no_hz.html#omit-scheduling-clock-ticks-for-cpus-with-only-one-runnable-task
+.. _kernel parameters: https://docs.kernel.org/admin-guide/kernel-parameters.html
+.. _cpu list: https://docs.kernel.org/admin-guide/kernel-parameters.html#cpu-lists
+.. _irqaffinity: https://docs.kernel.org/core-api/irq/irq-affinity.html
+.. _snap unset: https://snapcraft.io/docs/set-system-options
+.. _building a gadget snap: https://ubuntu.com/core/docs/gadget-building
+.. _pc-gadget: https://github.com/snapcore/pc-gadget
+.. _pi-gadget: https://github.com/snapcore/pi-gadget
+.. _using snap options: https://snapcraft.io/docs/system-options
```

Comment:
```suggestion
.. _ snap options: https://snapcraft.io/docs/system-options
```

Based on earlier comment on link text since this goes to the list of snap options.